test/format/wts.c

```diff
@@ -273,6 +273,17 @@ create_database(const char *home, WT_CONNECTION **connp)
     if (GV(DISK_DATA_EXTEND))
         CONFIG_APPEND(p, ",file_extend=(data=8MB)");
 
+    if (GV(DEBUG_REALLOC_EXACT))
+        CONFIG_APPEND(p, ",debug_mode=(realloc_exact=true)");
+
+    /* Configure realloc malloc debug mode. */
+    if (GV(DEBUG_REALLOC_MALLOC)) {
+        if (mmrand(NULL, 0, 1) == 1)
+            CONFIG_APPEND(p, ",debug_mode=(realloc_exact=true,realloc_malloc=true)");
```

Comment:
Do we need this setting of both of them? Or, given the above line adding `realloc_exact` at lines 277, if we want to add both here also please add a comment about this conditional and why we want one or both 50% of the time.